project/src/main/java/edu/depaul/cdm/se452/project/model/Professor.java

```diff
@@ -0,0 +1,55 @@
+package edu.depaul.cdm.se452.project.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import lombok.Data;
+import lombok.ToString;
+
+@Data
+@Entity
+@Table(name = "PROFESSORS")
+public class Student implements Serializable {
+	
+	private static final long serialVersionUID = 1L;
+
+	@Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int employee_id;

```

Comment:
I believe this should be employeeId.  The SQL queries used for Flyway require the column names to be like employee_id, but the code should be camel case.